src/Week2_HW.py

```diff
@@ -0,0 +1,141 @@
+import sys
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn.datasets import load_iris
+from sklearn.ensemble import HistGradientBoostingClassifier, RandomForestClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+
+
+def summary_stats(arr):
+
+    """
+
+    Function that returns pandas DF from
+    numpy statistics
+    """
+
+    mean = np.array(arr).mean()
+    numpy_max = np.max(np.array(arr))
+    numpy_min = np.min(np.array(arr))
+    quant = np.quantile(np.array(arr), [0.25, 0.5, 0.75])
+
+    # Creating report summary from numpy stats
+    # Need to wrap dict in a list to give DF an index
+    report = pd.DataFrame(
+        [
+            {
+                "Mean": mean,
+                "Max": numpy_max,
+                "Min": numpy_min,
+                "Quantiles (25%, 50%, 75%)": quant,
+            }
+        ]
+    )
+    return report
+
+
+def main():
+
+    # loading iris dataset from sklearn
+    iris = load_iris()
```

Comment:
Are we allowed to use load_iris()? I think it would be better to stick with iris.data file.

Reply:
That is some very sound advice!